### PR TITLE
expression: implement vectorized evaluation for `builtinBitNegSig`

### PR DESCRIPTION
**Pull request by AerysNan**
PCP #12103
### What problem does this PR solve? 
Implement vectorized evaluation for builtinBitNegSig

### What is changed and how it works?
```
BenchmarkVectorizedBuiltinOpFunc/builtinBitNegSig-VecBuiltinFunc-56              2170525               602 ns/op               0 B/op          0 allocs/op
BenchmarkVectorizedBuiltinOpFunc/builtinBitNegSig-NonVecBuiltinFunc-56             47398             28934 ns/op               0 B/op          0 allocs/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: expression/builtin_op_vec.go [hunk 1]
```diff
@@ -190,11 +190,19 @@ func (b *builtinUnaryMinusRealSig) vecEvalReal(input *chunk.Chunk, result *chunk
...
 
 func (b *builtinBitNegSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+	n := input.NumRows()
+	args := result.Int64s()
+	for i := 0; i < n; i++ {
+		args[i] = ^args[i]
```
Comment (Reminiscent) [inline expression/builtin_op_vec.go]: We should check ```input.IsNull(i)``` first.

Comment (qw4990) [reply expression/builtin_op_vec.go]: We can ignore this check to speed up since it doesn't affect the correctness of our results.

**Comment by AerysNan** [comment]
@sre-bot /run-all-tests

**Comment by qw4990** [comment]
> @Reminiscent Automerge failed. Maybe merge it again?

I will merge it manually soon.

**Pull request merged**